x-pack/plugins/security_solution/server/ui_settings.ts

```diff
@@ -112,6 +116,31 @@ export const initUiSettings = (uiSettings: CoreSetup['uiSettings']) => {
       requiresPageReload: true,
       schema: schema.boolean(),
     },
+    [DEFAULT_RULES_TABLE_REFRESH_SETTING]: {
+      name: i18n.translate('xpack.securitySolution.uiSettings.rulesTableRefresh', {
+        defaultMessage: 'Rules auto refresh',
+      }),
+      description: i18n.translate(
+        'xpack.securitySolution.uiSettings.rulesTableRefreshDescription',
+        {
+          defaultMessage:
+            '<p>Enables auto refresh on the all rules and monitoring tables, in milliseconds</p>',
```

Comment:
This is the first time I have seen HTML mixed in with a defaultMessage like this? Is this correct or are the `<p>` going to end up in the translation file?

Reply:
It looks like it's fine and they translate around it:

```ts
"xpack.securitySolution.uiSettings.defaultAnomalyScoreDescription": "<p>機械学習ジョブの異常がこの値を超えると、セキュリティアプリに表示されます。</p><p>有効な値:0 ～ 100。</p>",
```

So no changes with this PR needed, but that just feels weird and off to me and maybe in a follow up or something one of us should just fix these weird things going on.

Reply:
Yea, I followed the existing pattern in that file but then also saw the i18n documentation have instances of it.